src/pynxtools_raman_multiformat/rod/rod_reader.py

```diff
@@ -0,0 +1,156 @@
+
+from typing import List, Dict, Any, Union
+import numpy as np
+import gemmi # for cif file handling
+
+from pathlib import Path
+
+
+class RodReader():
```

Comment:
I would call this something else, like `RodParser`, to distinguish from the `reader` object in `reader.py`.

Reply:
all right, was thinking about this. Used your suggestion.